libs/shared/src/lib/components/markdown/components/markdown-editor/classes/bold-button.class.ts

```diff
@@ -0,0 +1,19 @@
+import { MarkdownSyntaxEnum } from '../enums';
+import { MarkdownButton } from '../interfaces/markdown-button.interface';
```

Comment:
```suggestion
import { MarkdownButton } from '../interfaces';
```

І так в кожному відповідному класі